dev/tools/gen_defaults/lib/template.dart

```diff
@@ -157,6 +157,9 @@ abstract class TokenTemplate {
     final Map<String, dynamic> shape = tokens[tokens['$componentToken.shape']!]! as Map<String, dynamic>;
     switch (shape['family']) {
       case 'SHAPE_FAMILY_ROUNDED_CORNERS':
+        if (shape['topLeft'] == 0 && shape['topRight'] == 0 && shape['bottomLeft'] == 0 && shape['bottomRight'] == 0) {
+          return 'const RoundedRectangleBorder()';
```

Comment:
While I like the optimization here, I think we need to remove the `const` as it may wind up generating warnings in the generated code (which in this case it will).
```suggestion
          return 'RoundedRectangleBorder()';
```

Reply:
done